schema-registry/rest-service/src/test/java/com/hortonworks/registries/schemaregistry/avro/AbstractAvroSchemaRegistryCientTest.java

```diff
@@ -44,20 +44,28 @@
  */
 public abstract class AbstractAvroSchemaRegistryCientTest {
 
-    @ClassRule
-    public static final DropwizardAppRule<RegistryConfiguration> DROPWIZARD_APP_RULE
-            = new DropwizardAppRule<>(RegistryApplication.class, ResourceHelpers.resourceFilePath("schema-registry-test.yaml"));
-
-    protected final String rootUrl = String.format("http://localhost:%d/api/v1", DROPWIZARD_APP_RULE.getLocalPort());
-    protected final Map<String, String> SCHEMA_REGISTRY_CLIENT_CONF = Collections.singletonMap(SchemaRegistryClient.Configuration.SCHEMA_REGISTRY_URL.name(), rootUrl);
+    protected static LocalSchemaRegistryServer localSchemaRegistryServer;
```

Comment:
Should we convert this class to a Util class, as all the functions in the class are stateless?